src/backend/storeManagers/sideload/library.ts

```diff
@@ -59,7 +60,7 @@ export function addNewApp({
     current[gameIndex] = { ...current[gameIndex], ...game }
   } else {
     current.push(game)
-    addShortcuts(app_name)
+    addShortcuts(game)
```

Comment:
we already have the gameInfo here, no need to call the function that looks for the gameInfo